Description: Set class path to ./haxe/std/ and /usr/share/haxe/.
Author: Jens Peter Secher <jps@debian.org>
Index: haxe-3.0~svn6707/main.ml
===================================================================
--- haxe-3.0~svn6707.orig/main.ml	2013-07-18 12:39:57.000000000 -0400
+++ haxe-3.0~svn6707/main.ml	2013-07-18 14:54:03.220102949 -0400
@@ -807,7 +807,7 @@
 	with
 		Not_found ->
 			if Sys.os_type = "Unix" then
-				com.class_path <- ["/usr/lib/haxe/std/";"/usr/local/lib/haxe/std/";"/usr/lib/haxe/extraLibs/";"/usr/local/lib/haxe/extraLibs/";"";"/"]
+				com.class_path <- ["./haxe/std/";"/usr/shre/haxe/std/";"./haxe/extraLibs/";"/usr/share/haxe/extraLibs/";"";"/"]
 			else
 				let base_path = normalize_path (Extc.get_real_path (try executable_path() with _ -> "./")) in
 				com.class_path <- [base_path ^ "std/";base_path ^ "extraLibs/";""]);
